webapp/components/survey/CategorySelector/CategorySelector.js

```diff
@@ -71,24 +61,33 @@ const CategorySelector = (props) => {
           {i18n.t('common.manage')}
         </button>
       )}
-      {showCategoriesPanel && !editedCategory && (
+      {showCategoriesPanel && !categoryUuidToEdit && (
```

Comment:
when you go from categories to category, does it recreate the categories slide animation? if yes, keep the categories panel mounted when you open the category. it should work

Reply:
ok